src/renderer/components/Modal.tsx

```diff
@@ -11,7 +11,7 @@ export default class Modal extends React.Component {
       <div className="Modal">
         <div className="Modal__Inner">
           {this.props.children}
-          <button onClick={this.props.onClose}>Cancel</button>
+          <button className="Modal__Close" onClick={this.props.onClose}>Cancel</button>
```

Comment:
Nice. I should really write a CSS style guide, but you've pretty much got it. Summary:

* Top level of every React component simply has the component's full name as its CSS class
* Markup inside the component is `ClassName__Whatever`
* For different states of the same component, make `m-blah` classes. (e.g. `Checkbox m-disabled`)
* For classes used on different kinds of elements, use `u-blah`, like `u-button`